### PR TITLE
Various UX cleanup from user testing sessions.

### PR DESCRIPTION
**Pull request by jgerigmeyer**
## Cute animal pic
_Because everyone likes pictures of animals._

wombat

## Link to Trello card (if applicable)
_If this PR relates to a Trello card, don't forget to reference this PR on the card as well._

https://trello.com/c/yqKk4D4Q

## Description
_The commit messages say what you did; this should explain why and how._
- [x] Description is in Trello card

Various pieces of cleanup coming out of user testing sessions.

## Impacted areas of website
_Provide URLs and/or a description of where to look._

- Capture changes modal, help text for directories is moved into tooltips:

Screenshot 2020-03-27 13 10 51

- GitHub users are sorted alphabetically
- Split "Make changes in Dev Org" and "Retrieve changes from Dev Org" into two steps

Screenshot 2020-03-28 14 59 19

- Progress bar no longer looks clickable
- Cannot create Test Org if there are no task commits
- "Create Org" btns in cards are primary when that step is active

Screenshot 2020-03-28 15 06 04

- "Review" -> "Test", "Reviewer -> "Tester", "Review Org" -> "Test Org"
- "Capture Changes" -> "Retrieve Changes"
- Success toasts disappear when another success toast is added
- Show "Check for Unretrieved Changes" as large btn when no known changes to retrieve

Screenshot 2020-03-28 15 08 49

---

## Reviewer tasks:

- [ ] Pulled branch and ran code locally (or viewed on preview deployment, if applicable)
- [ ] Reproduced the issue to review in the browser
- [ ] Reviewed code
- [ ] Updated Trello card (if applicable)

### CODE REVIEW

**Comment by jgerigmeyer** [comment]
[UX cleanup from User Testing](https://trello.com/c/yqKk4D4Q/162-ux-cleanup-from-user-testing)

File: metecho/api/jobs.py [hunk 1]
```diff
@@ -515,14 +515,19 @@ def populate_github_users(repository, *, originating_user_id):
...
+            sorted(
+                [
+                    {
+                        "id": str(collaborator.id),
+                        "login": collaborator.login,
+                        "avatar_url": collaborator.avatar_url,
+                    }
+                    for collaborator in repo.collaborators()
+                ],
+                key=lambda x: x["login"],
```
Comment (jgerigmeyer) [inline metecho/api/jobs.py]: @wlonk This appears to be sorting case-sensitive, where I think we want case-insensitive.

Screenshot 2020-03-28 13 59 38

Comment (jgerigmeyer) [reply metecho/api/jobs.py]: @wlonk Also, could we sort the list of `project.github_users`?

Comment (wlonk) [reply metecho/api/jobs.py]: The backend never sets `project.github_users`, so the preferred way would be for the frontend to sort it when it sets it.

Comment (jgerigmeyer) [reply metecho/api/jobs.py]: Ok, I can do that. I thought maybe sorting in the serializer would be simplest.

**Pull request merged**